tests/OSPSuite.SimModel.Tests/OSPSuite.SimModel.Tests.vcxproj

```diff
@@ -209,6 +209,7 @@
       <OptimizeReferences>true</OptimizeReferences>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <TargetMachine>MachineX64</TargetMachine>
+      <AssemblyDebug>true</AssemblyDebug>
```

Comment:
This allows to see the full stack trace of the Exception when performing unit tests in Release!
Otherwise it's truncated